packages/drivers/replay-driver/src/storageImplementations.ts

```diff
@@ -127,15 +127,16 @@ export class SnapshotStorage extends ReadDocumentStorageServiceBase {
         return this.docTree;
     }
 
-    public async read(blobId: string): Promise<string> {
-        return this.storage.read(blobId);
-    }
-
     /**
-     * {@inheritDoc @fluidframework/driver-definitions#IDocumentStorageService.readString}
+     *
+     * @deprecated - only here for back compat, will be removed after release
      */
-    public async readString(blobId: string): Promise<string> {
-        return this.storage.readString(blobId);
+    public async read(blobId: string): Promise<ArrayBufferLike> {
+        return this.storage.readBlob(blobId);
```

Comment:
Delete? there is already readBlob that does the same. If you indeed need read(), it should return base64 encoded string